front/src/modules/timesStops/hooks/useTimeStopsColumns.tsx

```diff
@@ -24,14 +24,14 @@ const timeColumn = (isOutputTable: boolean) =>
     isCellEmpty: ({ rowData }) => !rowData,
   }) as Partial<Column<TimeExtraDays | undefined, string, string>>;
 
-const fixedWidth = (width: number) => ({ minWidth: width, maxWidth: width });
-
 export const useTimeStopsColumns = <T extends TimeStopsRow>(
   tableType: TableType,
   allWaypoints: T[] = []
 ) => {
   const { t } = useTranslation('timesStops');
 
+  const fixedWidth = (width: number) => ({ minWidth: width, maxWidth: width });
```

Comment:
Why did you move this function inside the hook ? I think you can leave it outside, this way it won't be re-instantiated at each execution

Reply:
I don't remember, so sure.